master/internal/stream/trials.go

```diff
@@ -198,10 +219,10 @@ func (ts *TrialFilterMaker) MakeFilter() func(*TrialMsg) bool {
 	// Make a copy of the maps, because the filter must run safely off-thread.
 	trialIds := make(map[int]bool)
 	experimentIds := make(map[int]bool)
-	for id, _ := range ts.TrialIds {
+	for id := range ts.TrialIds {
```

Comment:
whoa, this looks really wrong.

I think in my version I was collecting index and discarding the value, but here you're collecting the index and ignoring the value.

Both are wrong, we should be ignoring the index and collecting the value, right?

Reply:
oh!  `ts` is a `TrialFilterMaker`, not a `TrialSubscriptionSpec`, so you are right.  You are iterating over keys in a map, not indices of an array.

mibad.